journal/updated/05-chapter-4.adoc

```diff
@@ -39,205 +39,205 @@ Bristol, Fourth month 30th, 1763.
 so that we had hard work to get from one place to another in one day.
 We had a meeting at Esgyrgoch on third-day, which was dull and heavy.
 On fourth-day we went to Llenynde, to Humphrey Owens`',
 whose wife is the daughter of our worthy friend John Goodwin,
-whom I have often heard called the "`Prince of Wales;`" we were glad of his company,
-he and his wife both having lived to a good old age; and although thus advanced in years,
+whom I have often heard called the "`Prince of Wales.`" We were glad of his company,
+he and his wife both having lived to a good old age. And although thus advanced in years,
 they are living members fresh and green.
 
 We stayed here two nights and had a meeting,
-in which I believe we should have been more favoured if all of us had been faithful;
-I could not forbear thinking that something was lacking from our worthy friend;
-yet said within myself, perhaps it is only because I should be glad to hear him;
-but after meeting to my satisfaction and confirmation,
-he told me that he was afraid of being in the way of strangers.
+in which I believe we would have been more favoured if all of us had been faithful.
+I could not forbear thinking that something was lacking from our worthy friend,
+yet said within myself, "`Perhaps it is only because I would be glad to hear him.`"
+But after meeting, to my satisfaction and confirmation,
+he confessed he had not spoken for fear of being in the way of the visiting ministers.
 "`O!`" said I unto him, "`what a pity it was, that you, who are a father,
 should withhold anything from a poor child, who was in such need of a little help.`"
-He said, he always used to be fearful and backward.
+He said he always used to be fearful and backward.
 However, he was open and free in conversation,
 and told us many things which had befallen him in his pilgrimage,
-and how wonderfully the Lord had made way for him, and been with him all along,
+and how wonderfully the Lord had made way for him and been with him all along,
 even unto this day.
-Although at one time he had not where to lay his head,
-he now by the favour of Providence, enjoys plenty of the things of this life,
+Although at one time he had nowhere to lay his head,footnote:[Luke 9:58]
+he now by the favour of Providence enjoys plenty of the things of this life,
 is freed from the cares of worldly business, and like a faithful steward,
 is ready when his Lord may be pleased to call him.
-Oh! says my spirit, that I, who have known something of the same trials,
+"`Oh,`" says my spirit, "`that I, who have known something of the same trials,
 and have also known the arm of the Lord to be made bare for my help,
 may also know a being kept in humility,
-and in a true sense of the goodness of God and of my own nothingness, that so over all,
-God may be glorified, who alone is worthy.
+and in a true sense of the goodness of God and of my own nothingness, so that over all
```

Comment:
I think this would sound better if we changed it to: **so that God may be glorified over all,** who alone is worthy!